tests/models/test_amp.py

```diff
@@ -173,3 +173,15 @@ def test_cpu_model_with_amp(tmpdir):
 
     with pytest.raises((MisconfigurationException, ModuleNotFoundError)):
         tpipes.run_model_test(trainer_options, model, on_gpu=False)
+
+
+def test_amp_with_apex():
```

Comment:
apex still needs the choice of O1, O2, etc... how is that being done right now? we used to have a flag for this.

also, call it:
amp_backend='native | apex'